.github/workflows/e2e-tests.yml

```
@@ -98,12 +98,16 @@ jobs:
     - name: Run all tests
       if: matrix.gutenberg == 'no'
       working-directory: projects/plugins/jetpack/tests/e2e
-      run: yarn test-e2e
+      run: yarn test-e2e --testPathIgnorePatterns=search
```

For the time being, let's go with running search E2E tests on all changes regardless of labels since the workflow doesn't easily accommodate reacting to labeling events.

```suggestion
      run: yarn test-e2e
```

Once a file-based E2E test runner is in place, we can revisit introducing search E2E as a separate step.

---

> since the workflow doesn't easily accommodate reacting to labeling events.

Yep this is true however we can see whether Search or Instant Search label exists or not. And also if we do want to run search tests with the other parts, we'd remove the following search tests running step.

I would say keep search tests run separately based on labels for now. And when file-based test runner introduce, we could adjust.